source/js/components/validation.js

```diff
@@ -14,16 +14,26 @@ const monthlyOtherAmountErrorMessage = document.querySelector(
 
 // Handler for "other amount" inputs
 function otherAmountInputValidation() {
-  oneTimeOtherAmountInput.addEventListener("blur", (e) => {
-    if (document.querySelector(".one-time-donation-amount-radio:checked")) {
-      inputValueCheck(oneTimeOtherAmountInput, oneTimeOtherAmountErrorMessage);
-    }
-  });
-  monthlyOtherAmountInput.addEventListener("blur", (e) => {
-    if (document.querySelector(".monthly-donation-amount-radio:checked")) {
-      inputValueCheck(monthlyOtherAmountInput, monthlyOtherAmountErrorMessage);
-    }
-  });
+  if (oneTimeOtherAmountInput) {
```

Comment:
added if statements here as well, as we found that this was also experiencing the same issue where it would throw an "undefined" error if we were not on the page that had these input fields